tests/Definition/rlog/pr.fish

```diff
@@ -3,7 +3,8 @@ set TESTS ""
 set TESTS "$TESTS""250,runSingleTest1 gtest_replication2 -\n"
 set TESTS "$TESTS""250,runClusterTest1 replication2_server - --dumpAgencyOnError true\n"
 set TESTS "$TESTS""250,runClusterTest1 replication2_client - --dumpAgencyOnError true\n"
-set TESTS "$TESTS""250,runClusterTest1 auto - --dumpAgencyOnError true --test tests/js/common/shell/shell-replicated-logs-cluster.js\n"
```

Comment:
testdefnition.txt syntax should be used. 
Which jenkins job invokes these?

Reply:
https://jenkins01.arangodb.biz/view/PR/job/rlog-matrix-pr/